templates/store/store.html

```diff
@@ -31,59 +31,55 @@ <h2>Big data</h2>
   </div>
 </div>
 
-<div class="p-strip is-shallow u-no-padding--bottom">
+<div class="p-strip--accent is-slanted">
   <div class="row">
-    <div class="u-equal-height">
-      <div class="col-6 p-card">
+    <div class="col-6">
       <div class="col-2 u-align--center">
-        <img src="https://assets.ubuntu.com/v1/96c61b0d-charm-icon.png" alt="Illustration of a Charm pictogram">
+        <img src="https://assets.ubuntu.com/v1/96c61b0d-charm-icon.png" alt="">
       </div>
       <div class="col-4 u-no-margin--left">
+        <h4 class="p-muted-heading">Charms</h4>
         <p>Charms are sets of scripts that simplify the deployment and management tasks of a service. They are regularly reviewed and updated.</p>
         <a href="{{ url_for('jaasstore.search', type='charm') }}" class="p-button--neutral">View all the charms</a>
       </div>
     </div>
-      <div class="col-6 p-card">
+      <div class="col-6">
       <div class="col-2 u-align--center">
-        <img src="https://assets.ubuntu.com/v1/d4ffa9bc-bundle-icon.png" alt="Illustration of a Bundle pictogram">
+        <img src="https://assets.ubuntu.com/v1/d4ffa9bc-bundle-icon.png" alt="">
```

Comment:
alt removed

Reply:
Because its a decorative image that adds no value to the content it accompanies.